.all-contributors.md

```diff
@@ -0,0 +1,7 @@
+This file is required by [All Contributors](https://allcontributors.org/) to work properly and must not be deleted. Although it is automatically updated with the list of contributors, the actual output used in production is in JSON format and located in the `.all-contributorsrc` file.
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<!-- prettier-ignore-end -->
```

Comment:
> Shouldn't these be listed _before_ the start? I am concerned that these comments will be replaced by the AllContributors generated content.

Reply:
No, see the All Contributors [documentation](https://allcontributors.org/docs/fr/bot/installation).